src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('table');
+const tbody = document.querySelector('tbody');
+const tr = tbody.querySelectorAll('tr');
+const row = [...tr];
```

Comment:
I think Misha did mean like:
```suggestion
const table = document.querySelector('table');
const tbody = document.querySelector('tbody');
const row = [...table.rows];
```